src/styles/common.css

```diff
@@ -51,6 +51,71 @@ header > h1 {
   margin-right: auto;
 }
 
+header > nav button {
+  background: transparent;
+  border: none;
+}
+
+.fa-bars {
```

Comment:
This is more just a question rather than suggestion, but I wonder how come we don't use the `menu.svg` and `signout.svg` icons from the `assets` folder, since local svg files might be easier to colorize and resize (e.g. for desktop view) compared to font awesome.

Reply:
I had problems with the icons earlier, getting those to buttons the way I wanted to. So Ulla suggested that I could change them from local files to font awesome. I personally found them to be easier to style as I was able to treat those like text instead of pics 😅